docs/errors/E0149.md

```diff
@@ -8,7 +8,7 @@
 }
 ```
 
-`do`-`while` loops, `if` statements, `for` loops, `while` loops, and `with`
+`do`-`while` loops, `if` statements, `for` loops, `while` loops, `with` and labelled
 statements require a statement or list of statements for the body. It is a
 syntax error to write a class as the body of such a statement:
```

Comment:
Thinking aloud: Hmm, I think it's weird to talk about the 'body' for a labelled statement. We could make a new pair of diagnostic types to make the wording smoother, but I don't know if that's worth it.

Reply:
I actually had the same line of thought, but I couldn't think of a word much better than 'body'. I thought about 'postfix', but maybe some users will find that more confusing